### PR TITLE
remove proper noun Ethereum from all locales for onlyAddTrustedNetworks

### LINKED ISSUE #10388: Update custom network notification wording
When adding a custom RPC, a notification at the top of the window states:
A malicious Ethereum network provider can lie about the state of the blockchain and record your network activity. Only add custom networks you trust.

Ideally this should be more generic and say: A malicious network provider can lie about the state of the blockchain and record your network activity. Only add custom networks you trust.

image

### PR DESCRIPTION
**Pull request by brad-decker**
Fixes: #10388

Explanation:  
Removes proper noun 'Ethereum' from the translation strings. Going to need help verifying all locales. Note that I did not remove anything from the `ko` translation because the word `Ethereum` did not appear in the string.

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]
LGTM!

File: app/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -1128,7 +1128,7 @@
     "description": "Return the user to the site that initiated onboarding"
   },
   "onlyAddTrustedNetworks": {
-    "message": "恶意的 Ethereum 以太坊网络提供商可以伪造区块链状态，并记录您的网络活动。只添加您信任的自定义网络。"
+    "message": "恶意的 以太坊网络提供商可以伪造区块链状态，并记录您的网络活动。只添加您信任的自定义网络。"
```
Comment (Gudahtt) [inline app/_locales/zh_CN/messages.json:1131]: When testing this in Google Translate, "Ethereum" still appears in the output :thinking:

**Review by Gudahtt — COMMENTED** [review]
We might want to consider removing some or all of these translations. We might be making them confusing/grammatically incorrect by removing this term.

Also some of them still seem to include _translations_ of Ethereum. The `ko` message still includes 'Ethereum' in the output when translated with Google Translate for example.

**Comment by brad-decker** [comment]
>We might want to consider removing some or all of these translations. We might be making them confusing/grammatically incorrect by removing this term.

Should we just go the safest route and remove them, waiting for lionbridge to update them in the future? cc @Gudahtt

**Comment by Gudahtt** [comment]
That's essentially what we've done in most other cases where the _meaning_ of the message has changed. Deleting them seems appropriate. If there were any where you felt reasonable confident the sentences till makes sense without 'Ethereum' we could make exceptions though.

File: app/_locales/zh_CN/messages.json (hunk 1 above)
Comment (brad-decker) [reply app/_locales/zh_CN/messages.json:1131]: I'm not getting the same results, 
image

Comment (brad-decker) [reply app/_locales/zh_CN/messages.json:1131]: malicious blockchains, tho.

Comment (Gudahtt) [reply app/_locales/zh_CN/messages.json:1131]: I think you're translating a different locale here (this is simplified Chinese, not Japanese)

Comment (brad-decker) [reply app/_locales/zh_CN/messages.json:1131]: Oops, i was on the wrong locale.

**Comment by brad-decker** [comment]
In the end I removed a handful of translations that just didn't seem correct or to match the original intent of the message, and kept a few. I removed the Korean translation as it included the word Ethereum in it, so we'll need a new translation for that locale. The same goes for the simplified Chinese.

I removed Japanese and Filipino translations because the google translate seemed off to me and I want to ensure we get accurate translations.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**